### DISCUSSION TITLE
Small error on the Releases page

### DISCUSSION THREAD

**Discussion by AlonsoLP** [ExpressLRS/ExpressLRS#3084] [General]
There is a small error on the Releases page, specifically in the latest release section, at the beginning of version 3.5.3:

ExpressLRS V3.5.3 (Latest)
Changes since ___3.5.3___:

3.5.3 should be 3.5.2. I indicate it here because I don't know if this small error is worthy of opening an Issue.

Thank you very much for your work.

Greetings.

**Comment by pkendall64** [comment]
Thanks for this information. I've fixed the release notes.
